### DISCUSSION TITLE
Building a Docker image

### DISCUSSION THREAD

**Discussion by chemradio** [Joshua-Riek/ubuntu-rockchip#453] [Q&A]
Hello! Self-taught not-an-engineer here.

I have a goal of running Docker on Orangepi5plus. One of the containers should have GPU-support (Chrome will be installed, i need hardware acceleration). I was thinking that I could both flash this OS to my OrangePI as a host OS and run the same OS in the Docker container to inherit GPU support.

As I know not much about... almost anything in programming please could someone point me in the right direction? Right now I'm thinking of building a Docker image out of this repo to run in a container. Not sure how to do this though. Could somebody help me?

My ultimate goal is to run Selenium+Chrome in this container to render complex animation-heavy web pages and record the screen them with ffmpeg. The main issue is without the GPU hardware acceleration I'm not able to render the animations smoothly, thus the recording ends up being choppy. This is done with the purpose of automating GFX-element production in TV.
Example: https://github.com/chemradio/animation_debug

**Comment by armoreno** [comment]
Hi @chemradio,
Did you be able to get running a Docker on OrangePi+ with GPU support?
